test/e2e_node/runner/remote/run_remote.go

```diff
@@ -470,18 +460,14 @@ type imageObj struct {
 	name         string
 }
 
-func (io imageObj) string() string {
-	return fmt.Sprintf("%q created %q", io.name, io.creationTime.String())
-}
-
 type byCreationTime []imageObj
 
 func (a byCreationTime) Len() int           { return len(a) }
 func (a byCreationTime) Less(i, j int) bool { return a[i].creationTime.After(a[j].creationTime) }
 func (a byCreationTime) Swap(i, j int)      { a[i], a[j] = a[j], a[i] }
 
-// Returns a list of image names based on regex and number of previous images requested.
-func getGCEImages(imageRegex, imageFamily string, project string, previousImages int) ([]string, error) {
+// Returns an image name based on regex and given GCE project.
+func getGCEImage(imageRegex, imageFamily string, project string) (string, error) {
```

Comment:
IIUC regular expression can be used to get a number of images and not the only one image, why do we remove this functionality?

Reply:
Though we get multiple images, but the function always return one from same project or image family. Same as your below question. Regex is designed to match all available images to avoid yaml updates. That should be the design purpose.

Reply:
I see now, because of `gceImages.images[shortName] = gceImage` we anyway use only one image, thank for clarification.